packages/js/test/plugins/auctionHouseModule/withdrawFromTreasuryAccount.test.ts

```diff
@@ -0,0 +1,73 @@
+import test, { Test } from 'tape';
+import { sol } from '@/types';
+import {
+  metaplex,
+  killStuckProcess,
+  createNft,
+  createWallet,
+} from '../../helpers';
+import { createAuctionHouse } from './helpers';
+
+killStuckProcess();
+
+test('[auctionHouseModule] withdraw from treasury account on an Auction House', async (t: Test) => {
+  // Given we have an Auction House with fee that equals 100 basis points and an NFT.
```

Comment:
```suggestion
  // Given we have an Auction House with fee that equals 10% and an NFT.
```

According to the code, that's `1000` basis points not `100`. Also, in the comments, let's make it percentages so its clearer.